doc/sphinx/source/recipes/recipe_sea_surface_salinity.rst

```diff
@@ -0,0 +1,202 @@
+.. _recipes_sea_surface_salinity:
+
+Sea Surface Salinity Evaluation
+===============================
+
+Overview
+--------
+
+This recipe compares the regional means of sea surface salinity with a
+reference dataset (ESACCI-SEA-SURFACE-SALINITY v1 or v2 by default).
+To do this, the recipe generate plots for the timeseries of each region and
+a radar plot showing the correlation of dataset and reference timeseries for
+each region during the time they both exists.
+
+Preprocessor requirements:
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The recipe is created in a way that should make possible (although is not
+tested) to use it for other variables and datasets, even for more that one at
```

Comment:
```suggestion
tested) to use it for other variables and datasets, even for more than one at
```

Reply:
if it hasn't been tested please don't mention it; or, to the very least, mention it at the bottom via a `NOTE`